### PR TITLE
chore: add entity data to auditlog storage

### PR DESCRIPTION
**Pull request by kentakozuka**
part of https://github.com/bucketeer-io/bucketeer/issues/804

### CODE REVIEW

File: pkg/auditlog/storage/v2/sql/auditlog/select_audit_log_v2.sql [hunk 1]
```diff
@@ -6,7 +6,9 @@ SELECT
     type,
     event,
     editor,
-    options
+    options,
+    entity_data,
+    previous_entity_data,
```
Comment (cre8ivejp) [inline pkg/auditlog/storage/v2/sql/auditlog/select_audit_log_v2.sql:11]:
```suggestion
    previous_entity_data
```

File: pkg/auditlog/storage/v2/sql/auditlog/insert_audit_log_v2.sql [hunk 2]
```diff
@@ -8,4 +8,6 @@ INSERT INTO audit_log (
     editor,
     options,
     environment_namespace
+    entity_data,
+    previous_entity_data,
```
Comment (cre8ivejp) [inline pkg/auditlog/storage/v2/sql/auditlog/insert_audit_log_v2.sql:12]:
```suggestion
    previous_entity_data
```

File: pkg/auditlog/storage/v2/sql/auditlog/insert_audit_log_v2.sql [hunk 3]
```diff
@@ -8,4 +8,6 @@ INSERT INTO audit_log (
     editor,
     options,
     environment_namespace
```
Comment (cre8ivejp) [inline pkg/auditlog/storage/v2/sql/auditlog/insert_audit_log_v2.sql:10]:
```suggestion
    environment_namespace,
```

**Comment by kentakozuka** [comment]
@cre8ivejp Fixed 👍

**Review by cre8ivejp — APPROVED** [review]
Thank you!

**Pull request merged**